boards/arm/nrf52840_pca10059/nrf52840_pca10059.yaml

```diff
@@ -9,7 +9,12 @@ toolchain:
   - gnuarmemb
   - xtools
 supported:
+  - adc
```

Comment:
Together with this, you need to add a configuration for the board in `tests/drivers/adc/adc_api`, otherwise the Shippable check will fail.
Adding [here](https://github.com/zephyrproject-rtos/zephyr/blob/master/tests/drivers/adc/adc_api/src/test_adc.c#L43):
```
defined(CONFIG_BOARD_NRF52840_PCA10059) || \
```
should be sufficient.

Reply:
has been added.